packages/example/pages/_app.tsx

```diff
@@ -15,7 +16,7 @@ function MyApp({ Component, pageProps }: AppProps) {
       <WalletProvider
         chains={chains}
         assetLists={assets}
-        wallets={wallets}
+        wallets={wallets + trustWallet}
```

Comment:
this should likely be an array instead, something like `[...wallets, trustWallet]`